JASP-Engine/JASP/R/networkanalysis.R

```diff
@@ -1528,6 +1544,26 @@ NetworkAnalysis <- function(jaspResults, dataset, options) {
 
 }
 
+.networkAnalysisCheckKnownErrors <- function(e) {
+
+  # NOTE: this fails if glmnet decides to translate their error messages one day.
```

Comment:
Well it would be nice to handle this inside of bootnet itself... That does not mean you are supposed to do anything about it, I am just complaining that you have to do this matching of strings.

Reply:
I fully agree. R has built-in support for custom errors, so you could throw "InputError" or "DataError". It's just that almost no R packages use that... So to understand what goes wrong we have to resort to string parsing :(